src/pyshark/tshark/output_parser/tshark_ek.py

```diff
@@ -34,25 +34,46 @@ def _extract_packet_from_data(self, data, got_first_packet=True):
 
         return data[start_index:linesep_location], data[linesep_location + 1:]
 
-
-def packet_from_ek_packet(json_pkt):
+def packet_from_ek_packet_new(json_pkt):
```

Comment:
I don't think you mean to rename this func

Reply:
I'm seeing this when trying to use this branch:
```
>>> import pyshark
>>> file = "http_1.pcap"
>>> cap = pyshark.FileCapture(file, use_ek=True)
>>> pkt = cap[0]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/andrew/pyshark-fixed/src/pyshark/capture/file_capture.py", line 72, in __getitem__
    self.next()
  File "/home/andrew/pyshark-fixed/src/pyshark/capture/file_capture.py", line 62, in next
    packet = self._packet_generator.send(None)
  File "/home/andrew/pyshark-fixed/src/pyshark/capture/capture.py", line 222, in _packets_from_tshark_sync
    packet, data = self.eventloop.run_until_complete(
  File "/usr/lib/python3.10/asyncio/base_events.py", line 649, in run_until_complete
    return future.result()
  File "/home/andrew/pyshark-fixed/src/pyshark/tshark/output_parser/base_parser.py", line 15, in get_packets_from_stream
    packet = self._parse_single_packet(packet)
  File "/home/andrew/pyshark-fixed/src/pyshark/tshark/output_parser/tshark_ek.py", line 21, in _parse_single_packet
    return packet_from_ek_packet(packet)
NameError: name 'packet_from_ek_packet' is not defined. Did you mean: 'packet_from_ek_packet_new'?
```